streaming/src/test/java/org/apache/spark/streaming/JavaAPISuite.java

```diff
@@ -1739,7 +1739,11 @@ public Integer call(String s) throws Exception {
   // Java arguments and assign it to a JavaDStream without producing type errors. Testing of the
   // InputStream functionality is deferred to the existing Scala tests.
   @Test
-  public void testSocketTextStream() {
+  public void testSocketTextStream(
+
+
```

Comment:
Yes, this is a good functionality to revive. Thanks Sean

On Fri, Feb 20, 2015 at 4:17 AM, Sean Owen <notifications@github.com> wrote:

> In streaming/src/test/java/org/apache/spark/streaming/JavaAPISuite.java
> <https://github.com/apache/spark/pull/2765#discussion_r25066204>:
>
> > @@ -1739,7 +1739,11 @@ public Integer call(String s) throws Exception {
> >    // Java arguments and assign it to a JavaDStream without producing type errors. Testing of the
> >    // InputStream functionality is deferred to the existing Scala tests.
> >    @Test
> > -  public void testSocketTextStream() {
> > +  public void testSocketTextStream(
> > +
> > +
>
> Undo this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/2765/files#r25066204>.
>